lib/l10n_utils/tests/test_base.py

```diff
@@ -184,6 +184,9 @@ def test_ftl_activations(self, render_mock):
         (["en-US", "de"], ["de", "en-US"], "de"),
         (["en-US", "de", "fr"], ["fr", "de"], "fr"),
         (["en-US", "de", "fr"], ["en-CA", "fr", "de"], "en-US"),
+        # A request for only inactive translations should default to 'en-US'. Bug 1752823
+        (["am", "an", "en-US"], ["mk", "gu-IN"], "en-US"),
+        (["am", "an"], ["mk", "gu-IN"], "an"),
```

Comment:
Nit: could do with a comment here to explain why `am` is not selected 
```suggestion
        (["am", "an"], ["mk", "gu-IN"], "an"),  # "am" is not a supported locale; "an" is the first valid locale available as a fallback
```